dashboards/ec2/ec2_application_load_balancer_detail.sp

```diff
@@ -133,9 +156,13 @@ query "aws_alb_graph_relationships" {
       aws_vpc_security_group sg,
       alb
     where
-      sg.group_id in (select jsonb_array_elements_text(alb.security_groups))
+      sg.group_id in
+      (
+        select
+          jsonb_array_elements_text(alb.security_groups)
+      )
 
-    -- security groups - edges
+    -- To security group (edge)
```

Comment:
```suggestion
    -- To VPC security groups (edge)
```

Please see formatting notes for the comment as per https://github.com/turbot/steampipe-utils/wiki/Resource-Relationship-Graph-Checklist#edges